Automattic/sensei
# Prefix email meta keys with underscore to make it private

Makes the `sensei_email_type` and `sensei_email_description` meta keys private to that custom fields don't show it.

### Changes proposed in this Pull Request

* Added '_' prefix to `sensei_email_type` and `sensei_email_description` meta keys

### Testing instructions

- Add a few sensei_email posts
- Update a few emails for students `update_post_meta( $post->ID, '_sensei_email_type', 'student' );`
- To the others add post meta sensei_email_type with the value teacher.
`update_post_meta( $post->ID, '_sensei_email_type', 'teacher' );`
- Add post meta `_sensei_email_description` with the value teacher.
`update_post_meta( $post->ID, '_sensei_email_description', 'Test description' );`
- Make sure they appear in the student and teacher tabs as expected
